tools/build.py

```diff
@@ -42,59 +42,74 @@ def __init__(self, build_target, l2tdevtools_path):
       l2tdevtools_path (str): path to l2tdevtools.
     """
     super(ProjectBuilder, self).__init__()
+    self._build_helpers = {}
     self._build_target = build_target
     self._l2tdevtools_path = l2tdevtools_path
+    self._source_helpers = {}
 
-  def _BuildProject(self, download_helper_object, project_definition):
-    """Builds a project.
+  def _BuildProjectForDistribution(
```

Comment:
Please update the docstring, and perhaps rename to method to indicate that this also builds projects for which there is no distribution.  Could you refactor to make distribution an optional arg, to make this clearer?

Reply:
This method is being added in this PR, so I don't know what you mean by "additional refactor". The refactor is happening here, now. 

I don't know what you mean about the semantics of distribution - per the docstring this is the "name of the distribution", and is  None for every build target except dpkg-source.

Reply:
ack, per conversation, which was not clear from the initial review comment, reverted the name change of the method